### DISCUSSION TITLE
0.6.0 is avaialable, but no record of a release on GH

### DISCUSSION THREAD

**Discussion by CodingDoug** [google/yggdrasil-decision-forests#119] [General]
Wanted to point out that [0.6.0 is released to pypi](https://pypi.org/project/ydf/) but there is no record of that release [here on GitHub](https://github.com/google/yggdrasil-decision-forests/releases).

**Comment by rstz** [comment]
You're absolutely right, I will fix this tomorrow (and see if there is an automated process that works for us)!

**Reply by rstz** [reply]
Fixed :)
